isy-serviceapi-core/src/test/java/de/bund/bva/isyfact/serviceapi/core/aop/StelltLoggingKontextBereitAspectTest.java

```diff
@@ -119,6 +122,9 @@ public void testAufrufLeereKorrelationsId() {
         assertFalse(korrId.isEmpty());
         //assert MDC state after call
         assertNull(MdcHelper.liesKorrelationsId());
+
+        // assert that generated correlation id was added to AufrufKontextTo
+        assertEquals(service.getKorrelationsIDLetzterAufruf(), to.getKorrelationsId());
```

Comment:
Missing the logic in the testcase `testAufrufMitException` in line 151 to check if the `korrelationsId` is removed from `aufrufKontextTo` in case of exception.

Reply:
This behaviour would only be expected if the AufrufKontextTo was cleared inside the finally block of the StelltLoggingKontextBereitInterceptor.

Reply:
That's correct and as the reanswer above describes this should be part of the finally block.